internal/api/github/client.go

```diff
@@ -264,21 +265,36 @@ func (c *Client) GetTeamMembers(ctx context.Context, teams []*github.Team, organ
 		logger.WithFields(log.Fields{"page": opts.Page}).Tracef("requesting")
 
 		ctxTimeout, fn := context.WithTimeout(ctx, DefaultGitHubOperationTimeout)
-		m, res, err := c.githubClient.Teams.ListTeamMembers(ctxTimeout, team.GetID(), opts)
+		org, resorg, err := c.githubClient.Organizations.Get(ctx, organisation)
+		if err != nil {
+			fn()
+			return nil, fmt.Errorf("error getting an organisation %q: %w", organisation, err)
+		}
+		if resorg.StatusCode >= 300 {
+			fn()
+			return nil, fmt.Errorf("error getting an organisation organisation %q (status: %d): %s", organisation, resorg.StatusCode, readAllClose(resorg.Body))
+		}
+		fn()
```

Comment:
I can see you're just copying the existing pattern with calling `fn()` in every branch so I'm happy to merge as is, but a few Qs:
1. Isn't this a prime case for using `defer`?
2. Why do we cancel the context (`fn()`) at all? The places that we do this, we've already exited all the function calls that we've passed `ctxTimeout` to.

Reply:
I didn't give it too much thought. As you noted, I just copy/pasted existing pattern. I wanted this PR to change as little as possible to fix the issue without making to much mess. I can take a closer look at it next week.